### ISSUE TITLE
UnboundLocalError: local variable 'command_line' referenced before assignment

### ISSUE THREAD

**Issue by chming1016** [tern-tools/tern#916]
**Describe the bug**
When trying to analyze the "percona/percona-backup-mongodb:1.3.4" image, tern gives errors with "UnboundLocalError: local variable 'command_line' referenced before assignment"

**To Reproduce**
tern -c report -i percona/percona-backup-mongodb:1.3.4 -f json -o output.json

**Error in terminal**
```
2021-03-17 19:00:30,020 - DEBUG - common - Reading files in filesystem...
Traceback (most recent call last):
  File "/root/.virtualenvs/test/bin/tern", line 8, in <module>
    sys.exit(main())
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/__main__.py", line 237, in main
    do_main(args)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/__main__.py", line 96, in do_main
    crun.execute_image(args)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/container/run.py", line 87, in execute_image
    cimage.analyze(full_image, args.redo, args.driver, args.extend)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/container/image.py", line 74, in analyze
    default_analyze(image_obj, redo, driver)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/container/image.py", line 64, in default_analyze
    image_obj, shell, master_list, redo, driver)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/container/multi_layer.py", line 99, in analyze_subsequent_layers
    fresh_analysis(image_obj, curr_layer, shell, driver, envs)
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/container/multi_layer.py", line 52, in fresh_analysis
    image_obj.layers[curr_layer])
  File "/root/.virtualenvs/test/lib/python3.6/site-packages/tern/analyze/default/default_common.py", line 119, in get_commands_from_metadata
    files=command_line), 'warning'))
UnboundLocalError: local variable 'command_line' referenced before assignment
```

**Expected behavior**
Tern should output the json report.

**Environment you are running Tern on**
- Output of 'tern --version'
d6ca178 Set SPDX PackageDownloadLocation to valid value
- Operating System (Linux Distro and version or Mac or Windows)
Ubuntu 18.04.3 LTS
- Container OS
Red Hat Enterprise Linux Server 7.8 (Maipo)
- Python version (3.6 or higher)
Python 3.6.8

**Please attach files if they exist**
- [tern.log](https://github.com/tern-tools/tern/files/6155806/tern.log)

**Comment by nishakm** [comment]
@chming1016 I cannot reproduce this error at the tip of main. Can you try with either version 2.4.0 or the tip of main?

**Comment by chming1016** [comment]
Hi @nishakm, 

I have solved this issue while upgrade to 2.4.1 (current main).
However, this issue still remains in 2.4.0.

Thanks.

**Comment by nishakm** [comment]
@chming1016 We'll be cutting a new release soon - 2.5.0. Please use that one once it is published.
